blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/BlueOceanWebURLBuilder.java

```diff
@@ -128,6 +129,8 @@ private BlueOceanWebURLBuilder() {
             if (blueResource instanceof BlueMultiBranchPipeline) {
                 return getOrgPrefix(item) + "/" + encodeURIComponent(((BluePipeline) blueResource).getFullName()) + "/branches";
             }
```

Comment:
@alvarolobato 

>There is a possible NPE that we should protect from when you call OrganizationFactory.getInstance().getContainingOrg(i) by definition you could get a null, in which case we should fall back to a no org url:

Good catch, there is NPE at couple places, lucky it didn't encounter so far because there is always default impl of `OrganizationFactory`. I will fix that.

> Maybe just getting the org and if not null return the org url instead of blue for all the cases and not only for ModifiableTopLevelItemGroup that could be done a level above or checking all the if branches.

Ah, yeah, it won't work with how its done. Let me look in to this.